examples/pipelines/fl_cross_silo_basic/config.yaml

```diff
@@ -24,7 +25,8 @@ federated_learning:
         type: uri_file
         mode: 'download'
         path: https://azureopendatastorage.blob.core.windows.net/mnist/processed/t10k.csv
-    - compute: cpu-cluster-westus
+    - name: Silo-2
```

Comment:
could you use the region as a name? It would be more catchy in a demo to see which region is lighting up in the pipeline

Reply:
Sure.